velox/exec/OutputBufferManager.cpp

```diff
@@ -34,6 +35,14 @@ std::weak_ptr<OutputBufferManager> OutputBufferManager::getInstance() {
   return instance_;
 }
 
+// static
+const std::shared_ptr<OutputBufferManager>&
+OutputBufferManager::getInstanceStrong() {
+  static const std::shared_ptr<OutputBufferManager> instance =
+      std::make_shared<OutputBufferManager>(Options());
+  return instance;
+}
```

Comment:
The `OutputBufferManager::getInstance()` should be implemented in terms of `getInstanceStrong()`. Something like this:
```c++
std::weak_ptr<OutputBufferManager> OutputBufferManager::getInstance() {
	return getInstanceStrong();
}
```
Otherwise it will break any existing consumer of Velox since `getInstance()` and `getInstanceStrong()` would return different instances.